integration-tests/base/cronjob.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     component: integration-tests
 spec:
   schedule: "@daily"
-  suspend: true
+  suspend: false
```

Comment:
@fridex  can we let the base be suspended and turn on explicitly in each environment.
in this way, we would have flexibility in turning on or off , without concerning of base .

Reply:
Makes sense :+1: Adjusted in ad5e104